Tools/autotest/common.py

```diff
@@ -893,6 +893,20 @@ def reboot_sitl_mav(self, required_bootcount=None):
                                        0)
         self.detect_and_handle_reboot(old_bootcount, required_bootcount=required_bootcount)
 
+    def send_cmd_enter_cpu_lockup(self):
```

Comment:
It could be interesting to do a test without the enforcement to ensure that we don't lockup with random value ! And it will add 0.001% of additional coverage 😆